fcrepo-kernel-impl/src/test/java/org/fcrepo/kernel/impl/FedoraResourceImplTest.java

```diff
@@ -395,6 +398,14 @@ public void testGetProperty() throws RepositoryException {
     }
 
     @Test
+    public void testSetURIProperty() throws URISyntaxException, RepositoryException {
+        final String prop = "premis:hasEventRelatedObject";
+        final String uri = "http://localhost:8080/rest/1";
```

Comment:
It's a concern if those values are URIs of things in the repo.

On Thursday, April 23, 2015, Andrew Woods <notifications@github.com> wrote:

> In
> fcrepo-kernel-impl/src/test/java/org/fcrepo/kernel/impl/FedoraResourceImplTest.java
> <https://github.com/fcrepo4/fcrepo4/pull/781#discussion_r28990896>:
>
> > @@ -395,6 +398,14 @@ public void testGetProperty() throws RepositoryException {
> >      }
> >
> >      @Test
> > +    public void testSetURIProperty() throws URISyntaxException, RepositoryException {
> > +        final String prop = "premis:hasEventRelatedObject";
> > +        final String uri = "http://localhost:8080/rest/1";
>
> I also wonder about passing in values as full URLs that do not pass
> through an idTranslator. Is this a concern?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fcrepo4/fcrepo4/pull/781/files#r28990896>.
>